package.json

```diff
@@ -82,5 +82,6 @@
       "react-app",
       "react-app/jest"
     ]
-  }
+  },
+  "packageManager": "pnpm@9.6.0+sha512.38dc6fba8dba35b39340b9700112c2fe1e12f10b17134715a4aa98ccf7bb035e76fd981cf0bb384dfa98f8d6af5481c2bef2f4266a24bfa20c34eb7147ce0b5e"
```

Comment:
?

Reply:
NVM or some other CLI app is adding this on my PR's haven't figured out how to turn off. Will look more into it today